keep-a-changelog.py

```
@@ -0,0 +1,219 @@
+
+
+def get_git_diff():
+    result = subprocess.run(["git", "diff", "--cached", "--ignore-all-space"], stdout=subprocess.PIPE)
+    return result.stdout.decode("utf-8").strip()
+
+
+def read_file(filename):
+    try:
+        with open(filename, "r") as file:
+            return file.read()
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)